packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -192,6 +214,10 @@ export default class CFDStore extends BaseStore {
     get dxtrade_companies() {
         return getDxCompanies();
     }
+    // eslint-disable-next-line class-methods-use-this
```

Comment:
```suggestion
    // eslint-disable-next-line class-methods-use-this
```
we should try to avoid adding this disable comments